workers/unity/Assets/Playground/Config/CubeTemplate.cs

```diff
@@ -29,13 +22,14 @@ public static Entity CreateCubeEntityTemplate(Coordinates coords)
                 .AddMetadata(entityType, WorkerUtils.UnityGameLogic)
                 .SetPersistence(true)
                 .SetReadAcl(WorkerUtils.AllWorkerAttributes)
-                .AddComponent(transform, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeColor, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeTargetVelocity, WorkerUtils.UnityGameLogic)
-                .AddComponent(launchable, WorkerUtils.UnityGameLogic)
-                .Build();
+                .AddComponent(launchable, WorkerUtils.UnityGameLogic);
+
+            entity = TransformSynchronizationEntityBuilderHelper.AddComponents(entity, WorkerUtils.UnityGameLogic,
```

Comment:
nit: variable name `entity` => `entityBuilder`

Reply:
I also considered this change, left out in case there was any particular reason the cube template used a different name to the player template